flexinput/src/main/res/drawable/ic_audio_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="?flexInputIconColor"
```

Comment:
this styling only works for 23+. If the other icons are like this that's fine, but should check the main discord app

Reply:
I think this makes it black for <23, which is how the other icons in flexinput work.